### PR TITLE
Add CVE-2022-22963

### PR DESCRIPTION
**Pull request by RTann**
Information based on https://tanzu.vmware.com/security/cve-2022-22963 and https://sysdig.com/blog/cve-2022-22963-spring-cloud/

### CODE REVIEW

File: pkg/vulnloader/nvdloader/manual.go [hunk 1]
```diff
@@ -297,6 +297,71 @@ var manuallyEnrichedVulns = map[string]*schema.NVDCVEFeedJSON10DefCVEItem{
...
+		},
+		Configurations: &schema.NVDCVEFeedJSON10DefConfigurations{
+			CVEDataVersion: "4.0",
+			Nodes: []*schema.NVDCVEFeedJSON10DefNode{
+				{
+					CPEMatch: []*schema.NVDCVEFeedJSON10DefCPEMatch{
+						{
+							Cpe23Uri:              `cpe:2.3:a:apache:spring\-cloud\-function\-core:*:*:*:*:*:*:*:*`,
+							VersionEndExcluding:   "3.2.3",
+							VersionEndIncluding:   "3.2.0",
```
Comment (connorgorman) [inline pkg/vulnloader/nvdloader/manual.go:332]: This should be start right?

File: pkg/vulnloader/nvdloader/manual.go [hunk 2]
```diff
@@ -297,6 +297,71 @@ var manuallyEnrichedVulns = map[string]*schema.NVDCVEFeedJSON10DefCVEItem{
...
+						},
+						{
+							Cpe23Uri:              `cpe:2.3:a:apache:spring\-cloud\-function\-core:*:*:*:*:*:*:*:*`,
+							VersionEndExcluding:   "3.1.7",
```
Comment (connorgorman) [inline pkg/vulnloader/nvdloader/manual.go:336]: this just means all right?

**Review by connorgorman — CHANGES_REQUESTED** [review]
see comment about start instead of end including

File: pkg/vulnloader/nvdloader/manual.go (hunk 2 above)
Comment (RTann) [reply pkg/vulnloader/nvdloader/manual.go:336]: Yeah, when only `VersionEndExcluding` is specified, it's implied < 3.1.7 is affected

**Review by connorgorman — APPROVED** [review]
with tests passing, looks good

**Pull request merged**